ws-nextjs-app/pages/[service]/homepages/handleHomepageRoute.ts

```diff
@@ -0,0 +1,119 @@
+import { GetServerSidePropsContext } from 'next';
```

Comment:
Small nitpick but could this folder be called 'homepage' instead?

Realise some of the others are pluralised, but we tend to refer to a homepage as singular.